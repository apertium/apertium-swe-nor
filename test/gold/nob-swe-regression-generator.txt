[AWA7hOPgO92I]
inte. [/option]
[/AWA7hOPgO92I]
[B9i8-sqBsyuY]
vilka aktiviteter. [/option]
[/B9i8-sqBsyuY]
[M_hpqOV05M0j]
mammas telefon. [/option]
[/M_hpqOV05M0j]
[SCBaXMWwe1nl]
inte [/option]
[/SCBaXMWwe1nl]
[iCRNj2geRoJM]
Här har det även etablerats litteraturpris och ordnats litteraturfestivaler [/option]
[/iCRNj2geRoJM]
