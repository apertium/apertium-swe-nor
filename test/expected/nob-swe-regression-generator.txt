[8VMBw4Lp7TA0#0] Här har det också upprättats litteraturpris och blivit ordnade litteraturfestivaler.
[/8VMBw4Lp7TA0]
[AWA7hOPgO92I#0] inte.
[/AWA7hOPgO92I]
[B9i8-sqBsyuY#0] vilka aktiviteter.
[/B9i8-sqBsyuY]
[M_hpqOV05M0j#0] mammas telefon.
[/M_hpqOV05M0j]
[dGFAyy7M9Fgp#0] Jag ser bara mina vänner virtuellt.
[/dGFAyy7M9Fgp]
[uXuhSTHKMz88#0] något annat land i världen.
[/uXuhSTHKMz88]
